### DISCUSSION TITLE
Create automation script for reboot once a day with 10 minute warning at specific time.

### DISCUSSION THREAD

**Discussion by BananamanCoolDude** [amidaware/tacticalrmm#1455] [General]
Good afternoon.

Could you guy tell me how to create an automation script for rebooting once a day with a 10-minute warning at a specific time?

Thanks. :)

**Comment by dinger1986** [comment]
using the command `shutdown.exe /F /R /T 600` will do what you want to achieve

**Comment by mearkats** [comment]
Schedule a reboot in ten minutes using powershell like the below

`shutdown -r -t $([int] ([datetime]"6PM"-(Get-Date)).TotalSeconds)`

Then you could use burnt toast, a powershell module to display a notification on screen to the user?

https://github.com/Windos/BurntToast
